cluster-autoscaler/metrics/metrics.go

```diff
@@ -390,20 +391,22 @@ var (
 		},
 	)
 
-	nodeGroupCreationCount = k8smetrics.NewCounter(
+	nodeGroupCreationCount = k8smetrics.NewCounterVec(
 		&k8smetrics.CounterOpts{
 			Namespace: caNamespace,
 			Name:      "created_node_groups_total",
 			Help:      "Number of node groups created by Node Autoprovisioning.",
 		},
+		[]string{"queued_provisioning"},
```

Comment:
IIRC `queued_provisioning` is a GCE/GKE-specific concept, right? These metrics are supposed to be cloud-provider-agnostic. WDYT about a type/kind (or other appropriate more generic term) label instead? And then cloud-provider-specific code can set the value to `queued_provisioning`?

Reply:
Changed to group_type.